src/main/java/com/exactpro/th2/hand/schema/CustomConfiguration.java

```diff
@@ -35,17 +37,17 @@ public class CustomConfiguration {
 
 	@JsonProperty(value="message-batch-limit")
 	private long messageBatchLimit = DEFAULT_MESSAGE_BATCH_LIMIT;
-	
-	@JsonProperty(value="driversMapping")
-	private Map<String, Map<String, String>> driversMapping;
-	
+
+	@JsonProperty(value="driversMapping", required = true)
+	private Map<String, Config.DriverMapping> driversMapping;
```

Comment:
Are you sure that it will be able to deserialize the original config? Please, check it and correct if it does not